### ISSUE TITLE
Put focus on Ok button for Tree Delete Dialog

### ISSUE THREAD

**Issue by nielslyngsoe** [umbraco/Umbraco-CMS#8945]
￼![image](https://user-images.githubusercontent.com/6791648/93867840-e6db9e00-fcc9-11ea-8c83-49eecb40950e.png)

This "Ok" button should receive focus when the dialog opens, so the user can press enter immediately.

It would also be nice to rename the button to "Delete", this makes it more clear for the user what the action is.

**Comment by prasanna4567** [comment]
Hey there! I wish to work on this. 

And I also found that the tag [<umb-confirm>](https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Web.UI.Client/src/views/content/delete.html) in which changes had to be made to fix this issue has been defined in another [repo](https://github.com/umbraco/OurUmbraco/blob/main/OurUmbraco.Site/umbraco/Views/components/umb-confirm.html) under Umbraco. Could someone please guide me on how I should proceed with this? 

PS:  I am a first-timer. :)

**Comment by abjerner** [comment]
Hi @lynnewritescode. I'd say go for it 👍 

We usually don't assign issues to community members. Your comment should be a sufficient indicator to others that you may be working on this 😉

**Comment by daniel-shuy** [comment]
I've created a PR (#9260) that indirectly fixes this issue.

@lynnewritescode Sorry if you're still working on this issue, I assumed not since its been 22 days

**Comment by lynnewritescode** [comment]
@daniel-shuy don’t worry, I haven’t been able to look at it!

**Comment by daniel-shuy** [comment]
@nathanwoulfe I hope its not too late, but is this eligible for the Hacktoberfest swag?
